parastell/parastell.py

```diff
@@ -392,29 +423,36 @@ def export_dagmc(
             'Exporting DAGMC neutronics model...'
         )
 
-        if self.invessel_build:
-            self._import_ivb_step()
-
-        if skip_imprint:
-            self.invessel_build.merge_layer_surfaces()
-        else:
-            cubit.cmd('imprint volume all')
-            cubit.cmd('merge volume all')
-
-        if legacy_faceting:
-            self._tag_materials_legacy()
+        if self.legacy_faceting:
             cubit_io.export_dagmc_cubit_legacy(
                 filename=filename,
                 export_dir=export_dir,
                 **kwargs
             )
         else:
-            self._tag_materials_native()
             cubit_io.export_dagmc_cubit_native(
                 filename=filename,
                 export_dir=export_dir,
                 **kwargs
             )
+    
+    def export_cub5(self, filename='dagmc', export_dir='', **kwargs):
```

Comment:
Again, I don't believe `kwargs` is necessary here.

Reply:
I think you are right